STEERING-COMMITTEE.md

```diff
@@ -97,60 +144,18 @@ first name):
 |  | Open \*      | Google       |  |
 
 
-\* Open seats have been allocated to the organization per the committee rules,
-  but have not been assigned to a member of the organization.
-
-### Allocation of seats
-
-Seats on the steering committee are allocated based upon contribution to the
-project by an organization. No final decision has been made on the exact
-formula.
-
-As the project continues to grow, we expect to add additional seats to the
-committee, ensuring that those contributing to the project are properly
-represented.
-
-- After a seat is allocated to an organization, the organization shall nominate
-  a candidate to be confirmed by KSC. The committee reserves the right to not
-  confirm a candidate, in which the organization would need to nominate a new
-  candidate
-- Members of the committee may step down at any time. When a member steps down,
-  their organization shall nominate a new candidate.
-- If a member leaves their organization, the organization must nominate a new
-  committee member to replace them following the nomination and confirmation
-  process.
-- If an organization is unable to seat a candidate, the KSC reserves the right
-  to reallocate the seat to another organization.
-- Changes to the number of seats, which company seats are allocated to, and
-  nominations to the committee are confirmed by majority vote of the committee
-  members.
-- In situations where the organization which holds a seat is no longer a viable
-  entity (e.g. merger, dissolution, bankruptcy) the KSC will make a decision on
-  how to reallocate that seat. Seats do not automatically transfer to any
-  organization.
-- Members on the committee have a 1 year term from their confirmation, and may
-  be reconfirmed to the committee at the end of their term.
+\* Open seats have been allocated to the organization per the prior committee rules,
+  but have not been assigned to a member of the organization or replaced with elected seats.
 
 ## Decision process
 
 The steering committee desires to always reach consensus.
 
```

Comment:
Maybe it's worth including some examples of scenarios when voting is required? For example

"Decisions requiring a vote include: issuing written policy, amending existing written policy, creating, removing, or modifying a working group, all spending, hiring, and contracting, official responses to publicly raised issues, or any other decisions that at least half the members present decide require a vote."

Reply:
Done.